tests/python_tests/common.py

```diff
@@ -221,23 +222,25 @@ def get_multinomial_max_and_min_token() -> GenerationConfig:
 
 def get_test_dataset() -> Tuple[List[str], List[GenerationConfig]]:
     prompts = [
-        "What is OpenVINO?",
-        "How are you?",
-        "What is your name?",
-        "Tell me something about Canada"
+        "What is OpenVINO?" * 20,
+        "How are you?" * 70,
+        "What is your name?" * 70,
+        "Tell me something about Canada" * 40
     ]
     generation_configs = [
         get_greedy(),
-        get_beam_search(),
+        get_greedy(), # get_beam_search(),
         get_greedy(),
-        get_beam_search()
+        get_greedy(), #get_beam_search()
     ]
     return (prompts, generation_configs)
 
 
 def get_scheduler_config(scheduler_params: dict = None) -> SchedulerConfig:
     scheduler_config = SchedulerConfig()
     scheduler_config.cache_size = 1
+    scheduler_config.enable_prefix_caching = True
+    scheduler_config.use_cache_eviction = True
```

Comment:
do we need to enable it in current tests? I suppose it's tested in tests/python_tests/test_cache_optimizations.py

Reply:
Removed